runtime/src/lib.rs

```diff
@@ -85,7 +85,7 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPallets,
+	AllPalletsWithoutSystemReversed,
```

Comment:
Are you sure about this?

Reply:
According to https://github.com/paritytech/substrate/pull/10043 this is the same as the old value. I updated the PR description.

Reply:
It seems that **frame_executive::Executive** has also changed:

frame_executive::Executive requires as generic all pallets with system. Before this PR the type required as generic all pallets without system and always executed system pallet hook before all pallets hook. But this requires duplicate works and error prone implementation thus it now requires all pallets with system. The previous order was to execute the hook on system pallet first and then AllPallets. It is recomended to use one of AllPalletsWithSystem or AllPalletsWithSystemReversed or AllPalletsReversedWithSystemFirst.

Don't you think it should be  **AllPalletsReversedWithSystemFirst** ?

Reply:
you are right. I changed it.